atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -291,6 +292,21 @@ def run(self):
         # label is present
         release_label = labels.LABEL_NAMES[Labels.LABEL_TYPE_RELEASE][0]
 
+        # Reserve build for isolated builds as well (or any build with supplied release)
+        user_provided_release = self.workflow.user_params.get('release')
```

Comment:
is this block needed? IMHO this should be incorporated into block at the end (or bellow), it already checks release existence, do magic when scratch build, etc..

Does this also miss update of dockerfile (it's in the last code block)?
```
dockerfile_labels[release_label] = release
```

Reply:
I would rather keep it like this, because WHEN release is user_provided all we want to do is just check if exists, and reserve build, and not do anything else  : append, update dockerfile (that is done in add_labels plugin)

Reply:
I see duplicated code, it's error prone because changes to isolated builds may be ommited again

Reply:
it is more clear to have user_provided block separated, as we want to just check build and reserve it

instead of making block bellow more complex by adding IF user_provided to share 2 lines